util/build_docs.py

```diff
@@ -49,16 +49,19 @@
     # Pre-generate register and hwcfg fragments from these files.
     "hardware_definitions": [
         "hw/ip/aes/data/aes.hjson",
-        "hw/ip/alert_handler/data/alert_handler.hjson",
+        # "hw/ip/alert_handler/data/alert_handler.hjson",
```

Comment:
Please add a comment somewhere what the commented-out pieces mean, or remove them.

Reply:
Done